coordinator/src/resources/transport-wrapper.ts

```diff
@@ -0,0 +1,12 @@
+import {ParentPrescription} from "../services/hl7-v3-prescriptions";
+import {ElementCompact, xml2js} from "xml-js";
+import * as fs from "fs"
+import * as path from "path"
+
+export const wrap = function (payload: ParentPrescription): ElementCompact {
+    const options = {compact: true}
+    const wrapperStr = fs.readFileSync(path.join(__dirname, "./ConvertWrapper.xml"), "utf8")
```

Comment:
No need to read this every time, can just be a top level constant

Reply:
Could avoid parsing too by making wrapperJson the constant but would then need to clone it each time